depscan/cli.py

```diff
@@ -1115,7 +1115,7 @@ def main():
     # render report into template if wished
     if args.report_template and os.path.isfile(args.report_template):
         utils.render_template_report(
-            jsonl_report_file=report_file,
+            vdr_file=vdr_file,
```

Comment:
@heubeck If there are no vulnerabilities, there will be no vdr file produced. I see you check `if vdr_file` in render_template_report, but is there a purpose to calling the function in this scenario? If the desire is to produce a report even if there are no vulnerabilities (which is reasonable and what I do for csaf), the other elements (metadata, components, dependencies, etc.) can still be loaded using the bom_file. I'll leave another comment in utils with further details.